crates/bevy_gltf/src/loader.rs

```diff
@@ -298,6 +298,7 @@ fn load_material(material: &Material, load_context: &mut LoadContext) -> Handle<
         LoadedAsset::new(StandardMaterial {
             albedo: Color::rgba(color[0], color[1], color[2], color[3]),
             albedo_texture: texture_handle,
+            shaded: !material.unlit(),
```

Comment:
Is `shaded` the standard name here? It feels like `lit` is a more direct version of `!unlit`.

Reply:
Probably not, I just used `shaded` because that was already in the engine. I can rename the symbol though.

Reply:
godot uses "shadeless" in its materials (which inspired this), unity uses "unlit", unreal uses "unlit", and gltf uses "unlit".

I generally don't like "negative bools", as that can result in double negatives. But given that we're calling this `StandardMaterial`, we should probably align with the crowd here and rename this to "unlit" (and adjust the forward.frag shader to use `# ifndef STANDARDMATERIAL_UNLIT` instead of `# ifdef STANDARDMATERIAL_SHADED`)

Reply:
Ok, I've changed the name in 3241fb8.